how-tos/maintain/migrate-to-the-new-role-system.md

```diff
@@ -45,49 +45,14 @@ Please check your agent type and version and read on or switch to the right docu
 {% endtabs %}
 
 # Migrate to the new role system
-
```

Comment:
Remove title right on top of this line ? it is redundant with the page title?

Reply:
@arnaudbesnier is the owner of this, let's leave this as it is and change the format for all pages in another PR